ui/hooks/useTokenDisplayValue.js

```diff
@@ -43,7 +43,8 @@ export function useTokenDisplayValue(
       // and a token object has been provided
       token &&
       // and the provided token object contains a defined decimal value we need to calculate amount
-      token.decimals &&
+      token.decimals !== null &&
+      token.decimals !== undefined &&
```

Comment:
the problem is related to the fact that in Javascript , the boolean expression of `0` return `false`.